src/composables/pools/useTokenApprovals.ts

```diff
@@ -90,13 +90,13 @@ export default function useTokenApprovals(
     try {
       state.init = true;
 
-      const tx = await sendTransaction(
-        getProvider(),
-        address,
-        ERC20ABI,
-        'approve',
-        [spender, amount]
-      );
+      const txBuilder = new TransactionBuilder(getSigner());
+      const tx = await txBuilder.contract.sendTransaction({
+        contractAddress: address,
```

Comment:
This API is nice, I like specifying each argument by name much more than the list.